ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/ProcessingIdentityHandler.java

```diff
@@ -185,4 +187,20 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
             throw new RuntimeException();
         }
     }
+
+    private APIGatewayProxyResponseEvent performIntervention(
+            APIGatewayProxyRequestEvent input,
+            UserContext userContext,
+            ClientRegistry client,
+            AccountInterventionStatus aisResult)
+            throws Json.JsonException {
+        var logoutResult =
+                logoutService.handleAccountInterventionLogout(
+                        userContext.getSession(), input, client.getClientID(), aisResult);
+        var redirectUrl = logoutResult.getHeaders().get(ResponseHeaders.LOCATION);
+        return generateApiGatewayProxyResponse(
+                200,
```

Comment:
should this be 302 redirect?

Reply:
No, it's the API response to the frontend, which then generates the 302